### PR TITLE
Standardize ACME instruction details

### LINKED ISSUE #6448: TLS handschake error | acme can't get a cert for domain
### Description

**What happened**:
Followed instruction:
https://goteleport.com/docs/getting-started/
and for TLS certs
https://goteleport.com/blog/letsencrypt-teleport-ssh/

DNS records: (renamed to teleport.com for this purpose)
A record connect.teleport.com
A record *.connect.teleport.com

First 20/24 hours everything was working as it should be. Except for this TLS error showing in the server log.
After some time i noticed that the server section on the teleport webpage showed **No Data Available**

Error:
```
ERRO [PROXY:SER] "proxy2021/04/14 10:15:58 http: TLS handshake error from <client_ip>:46010: acme can't get a cert for domain 636f6e6e6563742e616c6c63732e636c6f7564.teleport.cluster.local, add it to the proxy_service.public_addr, or use one of the domains: connect.teleport.com\n" utils/cli.go:304
ERRO [PROXY:SER] "proxy2021/04/14 10:15:58 http: TLS handshake error from <client_ip>:46008: acme can't get a cert for domain 636f6e6e6563742e616c6c63732e636c6f7564.teleport.cluster.local, add it to the proxy_service.public_addr, or use one of the domains: connect.teleport.com\n" utils/cli.go:304
ERRO [PROXY:SER] "proxy2021/04/14 10:19:17 http: TLS handshake error from <client_ip>:46022: acme can't get a cert for domain 636f6e6e6563742e616c6c63732e636c6f7564.teleport.cluster.local, add it to the proxy_service.public_addr, or use one of the domains: connect.teleport.com\n" utils/cli.go:304
ERRO [PROXY:SER] "proxy2021/04/14 10:19:17 http: TLS handshake error from <client_ip>:46020: acme can't get a cert for domain 636f6e6e6563742e616c6c63732e636c6f7564.teleport.cluster.local, add it to the proxy_service.public_addr, or use one of the domains: connect.teleport.com\n" utils/cli.go:304
```

Server config:
```
teleport:
  nodename: connect.teleport.com
  data_dir: /var/lib/teleport
  auth_servers:
  - 127.0.0.1:3025
  log:
    output: stderr
    severity: INFO
  ca_pin: sha256:<ca_pin>
auth_service:
  enabled: "yes"
  listen_addr: 0.0.0.0:3025
  public_addr: connect.teleport.com:3025
  cluster_name: connect.teleport.com
ssh_service:
  enabled: "yes"
  labels:
    teleport: server
  commands:
  - name: hostname
    command: [hostname]
    period: 1m0s
proxy_service:
  enabled: "yes"
  listen_addr: 0.0.0.0:3023
  web_listen_addr: 0.0.0.0:443
  tunnel_listen_addr: 0.0.0.0:3024
  public_addr: connect.teleport.com:443
  https_keypairs:
    - key_file: /etc/letsencrypt/live/connect.teleport.com/privkey.pem
      cert_file: /etc/letsencrypt/live/connect.teleport.com/fullchain.pem
  acme:
    enabled: "yes"
    email: info@teleport.com
```

node config:
```
teleport:
  nodename: node-100101
  data_dir: /var/lib/teleport
  auth_token: <auth_token>
  ca_pin: sha256:<ca_pin>
  auth_servers:
    - connect.teleport.com:443
  log:
    output: stderr
    severity: INFO
auth_service:
  enabled: no
ssh_service:
  enabled: yes
  labels:
    teleport: node
  commands:
  - name: hostname
    command: [hostname]
    period: 1m0s
proxy_service:
  enabled: no

```

### Server Details
- Teleport version (run `teleport version`): **Teleport v6.1.1 git:v6.1.1-0-geacd474d8 go1.15.5**
- Server OS (e.g. from `/etc/os-release`): **Ubuntu 18.04.5 LTS**

### Client Details
- Tsh version (`tsh version`): **Teleport v6.1.1 git:v6.1.1-0-geacd474d8 go1.15.5**
- Computer OS (e.g. Linux, macOS, Windows): **Ubuntu 20.04.2 LTS**
- Browser version (for UI-related issues): **Chrome**
- Installed via (e.g. apt, yum, brew, website download): **apt**

### Debug Logs
included

[teleport-log.txt](https://github.com/gravitational/teleport/files/6309327/teleport-log.txt)
[teleport-log-client.txt](https://github.com/gravitational/teleport/files/6309328/teleport-log-client.txt)

### PR DESCRIPTION
**Pull request by ptgott**
Our Getting Started guides often include instructions for
configuring Let's Encrypt and ACME before starting
Teleport, but not all of these instructions have the same
level of detail, and some are missing some context around
how Teleport uses ACME and why you need to open port 443 on
your Proxy Service host. This change adds an include that
spells out these instructions and invokes the include in the
appropriate guides.

The intention was to include as much relevant information within
the guides themselves to prevent the reader from having to
navigate to other pages.

Closes #6448

### CODE REVIEW

File: docs/pages/application-access/getting-started.mdx [hunk 1]
```diff
@@ -31,31 +31,17 @@ $ docker run -d -p 3000:3000 grafana/grafana
...
+(!docs/pages/includes/permission-warning.mdx!)
 
 Download the latest version of Teleport for your platform from our
 [downloads page](https://goteleport.com/teleport/download).
 
-Teleport requires a valid TLS certificate to operate and can fetch one automatically
-using Let's Encrypt [ACME](https://letsencrypt.org/how-it-works/) protocol.
-
 We will assume that you have configured DNS records for `teleport.example.com`
 and `*.teleport.example.com` to point to the Teleport node.
```
Comment (zmb3) [inline docs/pages/application-access/getting-started.mdx:40]:
```suggestion
and `*.teleport.example.com` to point to the Teleport proxy.
```

Right?

File: docs/pages/includes/acme.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+Let's Encrypt verifies that you control the domain name of your Teleport deployment by communicating with the HTTPS server listening on port 443 of your Teleport Proxy Service.
+
+You can configure Teleport to complete the Let's Encrypt verification process—called the ACME protocol—by running the following `teleport configure` command, where `tele.example.com` is the domain name of your Teleport cluster and `user@example.com` is an email address used for notifications (you can use any domain):
```
Comment (zmb3) [inline docs/pages/includes/acme.mdx:3]: This reads a little bit awkward to me. My understanding is that `teleport configure` doesn't actually configure anything, but rather spits out some configuration that you can then paste into your teleport.yaml.

The way this reads, it sounds like `teleport configure` is actually modifying the file for you.

File: docs/pages/includes/database-access/start-auth-proxy.mdx [hunk 3]
```diff
@@ -3,24 +3,13 @@ Download the latest version of Teleport for your platform from our
 installation [instructions](../../installation.mdx).
 
 Teleport requires a valid TLS certificate to operate and can fetch one automatically
-using Let's Encrypt [ACME](https://letsencrypt.org/how-it-works/) protocol. We
+using Let's Encrypt. We
```
Comment (zmb3) [inline docs/pages/includes/database-access/start-auth-proxy.mdx:6]: Strange place for a line break.

**Review by zmb3 — APPROVED** [review]

**Review by benarent — APPROVED** [review]

**Comment by russjones** [comment]
@ptgott Looks like this was approved, can we merge?

**Pull request merged**